### DISCUSSION TITLE
Suppress or change the Courses link within a course

### DISCUSSION THREAD

**Discussion by dlglin** [openwebwork/webwork2#1794] [Ideas]
As suggested in [https://webwork.maa.org/moodle/mod/forum/discuss.php?d=8118](url), there are cases where the `Courses` link may not be desirable within a course.  The use case that was suggested is when LTI is the only authentication method.

Some ideas on implementation:

- There are several links that take you to the WW root page: the `Courses` link in the menu, the `webwork` link in the breadcrumbs and the WW logo (and possibly others).  Are there scenarios where these might point to different things, or should the behaviour of all of these be controlled by the same configuration?
- I see a couple of options for how to handle this:
1 Add configuration variables for the URL that this should point to and the desired text. e.g. `$webwork_home_url="https://yourLMS.edu/` and `$webwork_home_url_text="Back to $LMS_name"`.
2 Add a configuration option to disable these links completely.

**Comment by omar-hijab** [comment]
Thank you Danny.

Since "Courses" is not course-specific, this change should kick in only when LTI authen is enabled across-the-board, not on a per-course basis? Disabling the links mentioned above keeps things LMS-centric.

(I'm just starting with Git and WW, so I'd rather not submit my hack.)

**Comment by Alex-Jordan** [comment]
Some installations enable authen_LTI globally, but then some faculty run
their course without even using an LMS at all.  So whatever customization
there is, I don't think it should be necessarily tied to something in the
LTI config.

**Comment by omar-hijab** [comment]
Good point. In my use case, I'm not letting anyone log into the WW server directly, and to only go through the LMS, in the hope this will lessen hand-holding. It's our first WW semester, we'll see...
